docs/usage.md

```diff
@@ -31,6 +33,11 @@
   * [`--unmapped`](#--unmapped)
   * [`--save_trimmed`](#--save_trimmed)
   * [`--save_align_intermeds`](#--save_align_intermeds)
+  * [`--save_pileup_file`](#--save_pileup_file)
+  * [`--epiread`](#--epiread)
+  * [`--save_snp_file`](#--save_snp_file)
+  * [`--soloWCGW_file`](#--soloWCGW_file)
+  * [`--assets_dir`](#--assets_dir)
```

Comment:
these parameters should also have standard values

Reply:
What does it mean? sorry for not understanding...

Reply:
nextflow will throw ugly warnings if a parameter is not initialised to a default value (such as `false`)

Reply:
Oh OK, so I can initialized all this parameters to true. or false...